data/map.txt

```diff
@@ -29656,7 +29656,7 @@ planet Trove
 
 planet Tundra
 	attributes "dirt belt" oil textiles
-	landscape land/dmottl1
+	landscape land/snow20
```

Comment:
I'd rather Tundra keep dmottl1 and we change whatever other planet has dmottl1 to snow20.